contracts/interfaces/0.8.x/IFilteredMinterDAExpRefundV0.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import "./IFilteredMinterV1.sol";
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title This interface extends the IFilteredMinterV1 interface in order to
+ * add support for generic project minter configuration updates.
+ * @dev keys represent strings of finite length encoded in bytes32 to minimize
+ * gas.
+ * @author Art Blocks Inc.
+ */
+interface IFilteredMinterDAExpRefundV0 is IFilteredMinterV1 {
+    /// Auction details updated for project `projectId`.
+    event SetAuctionDetails(
+        uint256 indexed projectId,
+        uint256 _auctionTimestampStart,
+        uint256 _priceDecayHalfLifeSeconds,
+        uint256 _startPrice,
+        uint256 _basePrice
+    );
+
+    /// Auction details cleared for project `projectId`.
+    /// At time of reset, the project has had `numPurchases` purchases on this
+    /// minter, with a most recent purchase price of `latestPurchasePrice`. If
+    /// the number of purchases is 0, the latest purchase price will have a
+    /// dummy value of 0.
+    event ResetAuctionDetails(
+        uint256 indexed projectId,
+        uint256 numPurchases,
+        uint256 latestPurchasePrice
+    );
+
+    /// Maximum and minimum allowed price decay half lifes updated.
+    event AuctionHalfLifeRangeSecondsUpdated(
+        uint256 _minimumPriceDecayHalfLifeSeconds,
+        uint256 _maximumPriceDecayHalfLifeSeconds
+    );
+
+    /// sellout price updated for project `projectId`.
+    /// @dev does not use generic event because likely will trigger additional
+    /// actions in indexing layer
+    event SelloutPriceUpdated(
+        uint256 indexed _projectId,
+        uint256 _selloutPrice
+    );
+
+    /// artist and admin have withdrawn revenues from refundable purchases for
+    /// project `projectId`.
+    /// @dev does not use generic event because likely will trigger additional
+    /// actions in indexing layer
+    event ArtistAndAdminRevenuesWithdrawn(uint256 indexed _projectId);
+
+    /// receipt has an updated state
+    event ReceiptUpdated(
+        address indexed _purchaser,
+        uint256 indexed _projectId,
+        uint256 _netPaid,
+        uint256 _numPurchased
+    );
+
+    /// returns latest purchase price for project `_projectId`, or 0 if no
```

Comment:
We made a design decision a while ago to not allow a base price of zero. It is documented here: https://github.com/ArtBlocks/artblocks-contracts/pull/382#discussion_r1025882588

Overall, not allowing zero just makes the entire system easier to rationalize about, one example being that a value of zero returned here is guaranteed to mean no purchases have been made 😄 

Not allowing "potentially free mints" seems like a reasonable constraint on this minter, especially since ~the same effect could be made by having a base price of 1 wei.